resource_group.go

```diff
@@ -0,0 +1,53 @@
+package cloudflare
```

Comment:
just a note that we already have a resource container concept in this library that may be a little confusing here (even though i know IAM had resource groups first 😛 ). i wonder if there is a way we can better differentiate the two as unrelated? in the future versions of the library (see https://github.com/cloudflare/cloudflare-go/blob/master/docs/experimental.md) we'll have a clearer line as the IAM component will likely live in it's own subpackage/namespace as opposed to the top level concepts so this may not be a thing to solve now.

Reply:
Oof, yeah - I think let's work on this when re-packing the IAM components. I'm not sure we'll be moving away from the `ResourceGroup` terminology any time soon (and had a discussion today about extending it, actually) but will brainstorm ways to reduce confusion in this package.